packages/apps/plugins/plugin-space/src/SpacePlugin.tsx

```diff
@@ -113,7 +109,7 @@ export const SpacePlugin = (): PluginDefinition<SpacePluginProvides> => {
               default:
                 return null;
             }
-          case 'tree--empty':
+          case 'tree--empty': // TODO(burdon): Why double-hyphen?
```

Comment:
For roles I've used BEM, in case a distinction should be made between a variation (`--`) and a child (`__`).